packages/core/test/feedback/ScreenshotButton.test.tsx

```diff
@@ -180,6 +180,11 @@ describe('ScreenshotButton', () => {
       const takeScreenshotButtonAfterCapture = queryByText('Take a screenshot');
       expect(takeScreenshotButtonAfterCapture).toBeNull();
     });
+
+    await waitFor(() => {
+      const takeScreenshotButtonAfterCapture = queryByText('Remove screenshot');
```

Comment:
L: Just fixed the name and double negation not null.

````suggestion
      const removeScreenshotButtonAfterCapture = queryByText('Remove screenshot');
      expect(removeScreenshotButtonAfterCapture).toBeTruthy();
````

Reply:
Good catch Krystof 🙇 
Updated with 59597a4c99fb50675f62eb7c5f7c3ccd9bf0a169